Client Scripts/Whitespace Validation/whitespaceValidation.js

```diff
@@ -0,0 +1,9 @@
+// Client Script to Validate Whitespaces
+var a = g_form.getValue('field_name'); // Get value from the field
+	var b = a.length;
+	for (var i=0; i<b; ++i) {
+    if (a.charAt(i) === ' ') {
```

Comment:
Why not use a regex instead of looping through every character?

Reply:
Anyway execution time would be same because regex will also perform same in background

Reply:
Hi @wiz0floyd , I have regex instead of looping character